assets/stylesheets/issue_view_columns.css

```diff
@@ -35,3 +35,19 @@
     padding: 5px !important;
   }
 }
+
+#sorting_criteria_box {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.capitalize-text {
```

Comment:
These are pretty generic names that can override information in the base redmine (in case they use the same classes/ids).

Try to find a more specific names (for classes for example, using a more complete hierarchy, e.g.

`#relations .view-columns .issue > td.buttons`

Look in the rest of this file for examples.